core/src/test/java/io/grpc/internal/AbstractManagedChannelImplBuilderTest.java

```diff
@@ -363,5 +363,10 @@ protected ClientTransportFactory buildTransportFactory() {
     public Builder usePlaintext(boolean value) {
       throw new UnsupportedOperationException();
     }
+
+    @Override
+    public Builder useTransportSecurity() {
```

Comment:
This and several other of the implementations seem unnecessary, now that ManagedChannelBuilder. provides a default.